certora/specs/SafeWebAuthnSignerProxy.spec

```diff
@@ -21,10 +22,35 @@ use rule timeoutChecker filtered { f -> f.contract == currentContract }
 use rule simpleFrontRunning filtered { f -> f.contract == currentContract }
 use rule noRevert filtered { f -> f.contract == currentContract }
 use rule alwaysRevert filtered { f -> f.contract == currentContract }
+*/
+using SafeWebAuthnSignerSingleton as SafeWebAuthnSignerSingleton;
+
+hook DELEGATECALL(uint g, address addr, uint argsOffset, uint argsLength, uint retOffset, uint retLength) uint rc {
+    // DELEGATECALL is used in this contract, but it only ever calls into the singleton.
+    assert (executingContract != currentContract || addr == SafeWebAuthnSignerSingleton,
+        "we should only `delegatecall` into the singleton."
+    );
+}
+
+/*
+Property 12. Proxy - Delegate Call Integrity (calls the Singleton)
```

Comment:
where is the list of properties, if we give numbers we should have the list someplace (properties.md or readme.md under certora folder)

Reply:
Under certora folder we have `properties.txt`